### DISCUSSION TITLE
Control from the phone block screen

### DISCUSSION THREAD

**Discussion by gloaysa** [music-assistant/support#382] [Feature requests and ideas]
This one is similar to https://github.com/music-assistant/support/issues/350 and first of all I'd like to thank you for this project, it's something I've been wishing to have for a long time.

As a user, I'd like to control my music from MA as I can do with other music services as Spotify or Plex from my phone's lock screen when the app is running in the background.

I'm aware this is not only on MA side but on the companion apps, but it would be a killer feature.

Personally, what I find harder for my wife to accept on playing everything through HA with automations is that she doesn't want to have to open HA app to control the music when, if we use Spotify directly, she could just do it from her lock screen or watch.

**Comment by OzGav** [comment]
Please monitor this discussion and contribute anything further there https://github.com/orgs/music-assistant/discussions/396
